src/harvester/__tests__/queries.ts

```diff
@@ -1,38 +1,9 @@
-import { didsWithClaims, gqlArray, gqlNumber, gqlString } from '../queries';
+import { didsWithClaims } from '../queries';
 
 describe('didsWithClaims', () => {
-  test('should retrieve a graphql ast object', () => {
+  test('should return a GraphqlQuery object', () => {
```

Comment:
This test should verify that the variables are the same that were passed to the query